Tribler/Core/Utilities/utilities.py

```diff
@@ -21,25 +20,20 @@
 logger = logging.getLogger(__name__)
 
 
-def validTorrentFile(metainfo):
-    # Jie: is this function too strict? Many torrents could not be downloaded
-    if not isinstance(metainfo, DictType):
-        raise ValueError('metainfo not dict')
-
-    if 'info' not in metainfo:
-        raise ValueError('metainfo misses key info')
-
-    if 'announce' in metainfo and not isValidURL(metainfo['announce']):
-        # Niels: Some .torrent files have a dht:// url in the announce field.
-        if not metainfo['announce'].startswith('dht:'):
-            raise ValueError('announce URL bad')
-
-    # http://www.bittorrent.org/DHT_protocol.html says both announce and nodes
-    # are not allowed, but some torrents (Azureus?) apparently violate this.
+def validate_torrent_nodes(metainfo):
+    """
+    Validate the list of nodes in the metainfo if such list exists.
 
-    # if 'announce' in metainfo and 'nodes' in metainfo:
-    #    raise ValueError('both announce and nodes present')
+    First it is checked whether metainfo contains a list of nodes.
+    After this, for each node in the list, the following is checked:
+        - Whether the node is a pair of length two
+        - Whether the first element is of type String
+        - Whether the second element is a tuple consisting of an Int and a Long
 
+    :param metainfo: the metainfo for which the nodes have to be validated
+    :return: Returns the nodes of the metainfo or None if they don't exist
```

Comment:
Would you have a suggestion as to what to use as a general name for these methods? I could not think of anything better than validate. Moreover, one could read the docstring to know what the function returns.